Project.toml

```diff
@@ -6,6 +6,8 @@ version = "1.1.0"
 [deps]
 DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
 DiffEqSensitivity = "41bf760c-e81c-5289-8e54-58b1f1f8abe2"
+DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
```

Comment:
just use StochasticDiffEq. It's already imported.

Reply:
I removed the DifferentialEquations from dependencies.